src/redis-cli.c

```diff
@@ -2607,6 +2609,17 @@ static int isSensitiveCommand(int argc, char **argv) {
                 return 0;
             }
         }
+    } else if (argc > 4 && !strcasecmp(argv[0], "sentinel")) {
+        /* SENTINEL CONFIG SET sentinel-pass */
+        if (!strcasecmp(argv[1], "config") && !strcasecmp(argv[2], "set") && !strcasecmp(argv[3], "sentinel-pass")) {
+            return 1;
+        }
+        /* SENTINEL SET <mastername> auth-pass */
+        else if (!strcasecmp(argv[1], "set") && !strcasecmp(argv[3], "auth-pass")) {
```

Comment:
Is rename-command and auth-user need hidden?

Reply:
i suppose you're right.
in AUTH and HELLO we call redactClientCommandArgument for both user and pass.

Reply:
why `rename-command`?

Reply:
`rename-command`  used to ban come command in normal conditions, if `rename-command` write to the file  `~/.rediscli_history` maybe got by a man with evil intentions. I think its not safe.

Reply:
well, if you are the one that renamed a command (e.g. renamed `FLUSHALL` to `__FLUSHALL__`), you are also likely to be the one who uses it, so when you'll call `__FLUSHALL__`, that call will be logged in your history.
i don't think we wanna keep thinking of rename-command as a security feature, we aim to deprecate it.
that said, i don't mind hiding it either. so i'll delegate that decision to others / Yossi.

Reply:
> Is rename-command and auth-user need hidden?

you mean the sentinel-user or something else?  Please confirm it.  Thanks a lot

Reply:
> i suppose you're right. in AUTH and HELLO we call redactClientCommandArgument for both user and pass.

Let me add Hello option here. Thanks

Reply:
> i suppose you're right. in AUTH and HELLO we call redactClientCommandArgument for both user and pass.

Oh, yes, Hello is already there.